edx_credentials_themes/static/edx.org/certificates/sass/_print-rendering.scss

```diff
@@ -28,26 +27,41 @@
         box-shadow: none;
         padding: 0.625rem 2.5rem !important;
         background-color: $white;
+        height: 650px;
```

Comment:
Curious -- was there any reason we specifically set a height limit here? Did you run into some sort of issue without it?

Reply:
This prevents the "second blank page" issue. Did you solve this a different way?

Reply:
Ah! Yes, I solved it a slightly different way...
```
  html,
  body {
    margin: 40 10 10 10 !important;
    padding: 0 !important;
    height: auto;
    overflow: hidden; // without this, when printing the certificate it will include a blank white page
    background-color: $white;
  }
```

Specifically the line with `overflow` set to `hidden`.

Reply:
Ahh! thats probably a much better way to do it! I'm going to try and match it.

Reply:
🔧 'd